am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/RevokeAccessIntegrationTest.java

```diff
@@ -27,18 +39,111 @@ void setupTest() {
         resourceId = UUID.randomUUID().toString();
     }
 
+
     @Test
-    void whenRevokingResourceAccessResourceAccessRemovedFromDatabase() {
+    void whenRevokingResourceAccessShouldRemoveFromDatabase() {
         ams.grantExplicitResourceAccess(createGrantForWholeDocument(resourceId, READ_PERMISSION));
         ams.revokeResourceAccess(createMetadata(resourceId));
 
         assertThat(databaseHelper.countExplicitPermissions(resourceId)).isEqualTo(0);
     }
 
+    @Test
+    void whenRevokingResourceAccessOnRootShouldRemoveFromDatabase() {
+        ams.grantExplicitResourceAccess(createGrantForWholeDocument(resourceId, READ_PERMISSION));
```

Comment:
Any reason you did not do `grantExplicitResourceAccess(resourceId, "");` here as well for consistency?

Reply:
👍